draft-yasskin-http-origin-signed-responses.md

```diff
@@ -718,6 +753,10 @@ this leaks extra information.
 For non-executable resource types, a signed response can improve the privacy
 situation by hiding the client's interest from the original author.
 
+To prevent network operators other than `o1.com` or `o2.com` from learning which
+packages were read, clients SHOULD only load packages fetched over a transport
+at least as secure as TLS ({{?I-D.ietf-tls-tls13}}).
```

Comment:
1) Why not MUST?
2) How do you determine "as secure as". What is the guidance meant to be provided to implementations in determining that assessment? Is HTTP on a local network 'at least as secure as' HTTPS, if it's over a VPN?